src/mqt/bench/benchmark_generation.py

```diff
@@ -55,30 +55,33 @@ def generate_filename(
     benchmark_name: str,
     level: str,
     num_qubits: int | None,
+    is_mirror: bool = False,  # New parameter
```

Comment:
This should probably be named `generate_mirror_circuit`
And it should probably be the last parameter in the argument list.